src/views/guests/Register.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <div class="register">
+    <div class="registerdiv">
+      <br />
+      <h1 class="registerh1">Register</h1>
+      <input
```

Comment:
`action` attribute'u olmayan bir `form` tüm `input`'ları kaplamalı ve `submit`  tetikleniğinde `register` fonksiyonu çalışmalı.

Tarayıcı üzerinde JS çalışması kapatılabiliği için tüm inputlara `require` attribute'u eklenmeli.